features/org.wso2.carbon.deployment.engine.feature/resources/p2.inf

```diff
@@ -0,0 +1,2 @@
+instructions.configure = \
+org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../deployment); \
```

Comment:
we would need to create two folders, i.e., one for the Server and then the other for the Runtime. Also can we check in windows, whether there is any issue if it tries to create the folder, if the folder already exists.